### ISSUE TITLE
Bug: Golang project not getting PRs.

### ISSUE THREAD

**Issue by roopakv** [dependabot/feedback#679]
We had setup dependabot for a private golang project ~1 month ago and about 2 weeks ago we stopped getting PRs. Unsure how to fix this. I've tried clicking check now / Bump now on the dependabot site but it checks and nothing happens. Would love to figure out how to reset the connection so we start getting PRs again

**Comment by feelepxyz** [comment]
@roopakv 👋 which repo you seeing this on? If it's a private repo please email it to [support@dependabot.com](mailto:support@dependabot.com).

**Comment by roopakv** [comment]
@feelepxyz thank you for the response. Sent an email :)

**Comment by roopakv** [comment]
Closing this out. Thank you for the debugging help from dependabot folks. For anyone who lands here, here is what we figured out

- Our repo used go 1.13 (not an issue)
- Dependabot doesnt use go 1.13 which means they dont have the GOPROXY variable set by default
- We used a package which isnt available anymore `git.apache.org/thrift.git`

Resolution:
Move to github thrift.

Feature requests:
- Open issue on our repo instead of silently failing
